setup.py

```diff
@@ -36,5 +87,7 @@
         'dev': dev_requires,
     },
     packages=find_packages(),
+    ext_modules=[CMakeExtension('libpyg')],
```

Comment:
A little confused about naming it `libpyg` but in python having `torch.ops.pyg`, and folder name `pyg_lib`. Is there a way to make them look consistent?

Reply:
I currently do not see an easy way to do this.
* `lib*.so` is the standard naming of libraries
* `pyg` is the namespace
* `pyg_lib` is the Python package name.

I added a TODO here which we can look into later. Hope that works for you :)